charts/netbox/values.schema.json

```diff
@@ -1060,7 +1060,7 @@
               "type": "boolean"
```

Comment:
```suggestion
              "type": ["boolean", "string", "array"]
```

Reply:
I don't see, in what case this may be a String. I do however agree, that we should pick up array as an option.